scripts/gulp/dev-server.js

```diff
@@ -3,19 +3,70 @@
 const fs = require('fs');
```

Comment:
If the dev server is becoming a more fully-featured thing perhaps we should move it to `src/dev-server` or `scripts/dev-server` in future? It isn't really tied to gulp in any way.